raspberry/estimators/altitude_Kalman_filter.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+class KalmanFilter:
+    def __init__(self, sampling_frequency=10, x_tilde_0=np.array([[0,]*4]).T, P_0=np.array([[1, 1, 1, 1],]*4), cache_values=False) -> None:
+        self.__fs = sampling_frequency
+        self.__Ts = 1/self.__fs
+
+        # Normalized Throttle reference, range [0, 1]
+        self.__Tt = 0.
+        # System dynamics
+        self.__At = self.__update_At()
```

Comment:
`self.__At` should not be defined before you receive the first measurement

Reply:
Fixed it.